content/en/blog/2023/jaeger-exporter-collector-migration.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Migrating away from the Jaeger exporter in the Collector
+linkTitle: Migrating away from the Jaeger exporter in the Collector
+date: 2023-09-14
+author: >-
+  [Alex Boten](https://twitter.com/codeboten) (Lightstep)
```

Comment:
I think so far we always linked github accounts.

Reply:
I just followed the example i copied here: https://github.com/open-telemetry/opentelemetry.io/blob/1fb0f484fbac6df101139f7db789fbb306eb0f6e/content/en/blog/2023/outreachy-may-cohort.md?plain=1#L6

I don't have a problem changing it though.

Reply:
Ah... I missed that one. Whatever you prefer, I personally prefer the github accounts because (a) they give people direct insights into your work with OpenTelemetry and (b) your twitter link only works for people authenticated with the platform

Reply:
np. I like the idea of being consistent :)